### PR TITLE
Move integration tests to subdirectory, create API stubs

### PR DESCRIPTION
**Pull request by m-cat**

### CODE REVIEW

**Comment by m-cat** [comment]
I switched the export strategy to follow what seems like more idiomatic Python:

https://github.com/psf/requests/blob/master/requests/__init__.py

By getting rid of the Skynet class I was also able to separate out each API feature into its own file as well as remove all the `@staticmethod` annotations.

**Comment by m-cat** [comment]
TODO: refactor the options. There should be a `__default_options` function that takes the API endpoint and returns the options for that endpoint -- this is what we're doing in Go.

File: siaskynet/convert.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+"""
+
+
+from . import utils
+
+
+def __default_convert_options():
+    """Returns the default convert options."""
+
+    obj = utils.__default_options("/skynet/skyfile")
```
Comment (kwypchlo) [inline siaskynet/convert.py:11]: is this correct endpoint?

File: siaskynet/portals.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def get_portals(custom_opts={}):
+    """Returns the list of hashed merkleroots that are portalsed."""
+
+    raise NotImplementedError
+
+
+def update_portals(additions, removals, custom_opts={}):
+    """Updates the list of skylinks that should be portalsed from Skynet. \
```
Comment (kwypchlo) [inline siaskynet/portals.py:31]: `portalsed` sounds weird

File: siaskynet/convert.py (hunk 1 above)
Comment (m-cat) [reply siaskynet/convert.py:11]: Yeah, this is a double-purpose endpoint both for uploading and converting.

**Pull request merged**